lib/fizzy/parser_expr.cpp

```diff
@@ -27,6 +27,18 @@ inline void push(bytes& b, T value)
     b.append(storage, sizeof(storage));
 }
 
+template <typename T>
+inline parser_result<T> read(const uint8_t* pos, const uint8_t* end)
```

Comment:
I would call it maybe `read_value` or `read_const`

Reply:
Or `read_float`.